pkg/tasks/probes.go

```diff
@@ -174,10 +176,10 @@ func runProbes(s *state.State) error {
 		return nil
 	}
 
-	gteKube121Condition, _ := semver.NewConstraint(">= 1.21")
+	gteKube124Condition, _ := semver.NewConstraint(">= 1.24")
```

Comment:
If I'm not mistaken, this is doing defaulting. As we discussed, we want to containerd by default for new 1.21+ clusters, so this shouldn't be changed.

I'll double check and get back to you.